front/components/app/blocks/Data.tsx

```diff
@@ -82,6 +83,17 @@ export default function Data({
               readOnly={readOnly}
             />
           )}
+          {block.spec.dataset && (
+            <Button
+              variant="secondary"
+              onClick={() => {
+                window.location.href = `/w/${owner.sId}/a/${app.sId}/datasets/${block.spec.dataset}`;
+              }}
+              icon={PencilSquareIcon}
+              label="Edit"
```

Comment:
Do we want the modal here as well?

Reply:
Wdyt? I think it's less necessary - `data` block seems less likely to need quick iteration that `input` block, but we can add as well

Reply:
Just create two different experience for something that look exactly the same?

Reply:
What do we do about this paragraph then? Do I just add a flag to remove it?

<img width="615" alt="Screenshot 2024-05-02 at 12 44 23" src="https://github.com/dust-tt/dust/assets/1189312/726fefd4-b28d-4e76-985c-776c1f22fff3">